src/pages/ManagePath.js

```diff
@@ -91,6 +91,10 @@ const ManagePath = () => {
     setWordToDelete(null);
   };
 
+  const EditWord = (wordEntry) => {
+    navigate(`/uusisana/${wordEntry.pathId}`, { state: { wordEntry } });
+  };
```

Comment:
Kirjoita funktion nimi pienellä (en normaalisti jaksais valittaa näin pienistä mut menee samalla kun tekee ton yhe toisen asian :D)

Reply:
Mitäköhän tässäkin oli taas käynyt :D